src/EnergyPlus/FluidCoolers.cc

```diff
@@ -719,17 +514,16 @@ namespace FluidCoolers {
             }
         } else { // Fluid cooler performance input method is not specified as a valid "choice"
             ShowSevereError(cCurrentModuleObject + "= \"" + AlphArray(1) + "\", invalid " + cAlphaFieldNames(4) + " = \"" + AlphArray(4) + "\".");
-            ShowContinueError("... must be \"UFactorTimesAreaAndDesignWaterFlowRate\" or \"NominalCapacity\".");
+            ShowContinueError(R"(... must be "UFactorTimesAreaAndDesignWaterFlowRate" or "NominalCapacity".)");
```

Comment:
I assume this compiled...but what is the R for at the beginning of this string (ShowContinueError(R"(...)"?

Reply:
This is a way to do string literals where everything is taken exactly as it is in the string, no escaping.  It's nice because it allows quotes to be used in the string a bunch of places without worrying about having to escape each individual one.